3_plot/src/plot.R

```diff
@@ -0,0 +1,46 @@
+library(whisker)
+
+plot_figure <- function(summary_file, outfile, diagnostic_file){
+png(file = file.path(outfile), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
could you do a cmd-a, cmd-i on this file (and any others that need it) to use RStudio's default indentation? would make it easier to read.

Reply:
done